python/s3-sns-sqs-lambda-chain/cdk.json

```diff
@@ -0,0 +1,30 @@
+{
+  "app": "python3 app.py",
+  "watch": {
+    "include": [
+      "**"
+    ],
+    "exclude": [
+      "README.md",
+      "cdk*.json",
+      "requirements*.txt",
+      "source.bat",
+      "**/__init__.py",
+      "python/__pycache__",
+      "tests"
+    ]
+  },
```

Comment:
```suggestion
```
For consistency with the other examples let's try to keep this config light by only including the required configurations.